x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -80,25 +79,64 @@ describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
       .should('have.text', createdHost.hostname);
   });
 
+  it('should update endpoint policy on Endpoint', () => {
+    const parseRevNumber = (revString: string) => Number(revString.match(/\d+/)?.[0]);
+
+    loadPage(APP_ENDPOINTS_PATH);
+
+    cy.getByTestSubj('policyListRevNo')
+      .first()
+      .invoke('text')
+      .then(parseRevNumber)
+      .then((initialRevisionNumber) => {
+        // Update policy
+        cy.getByTestSubj('policyNameCellLink').first().should('exist').click();
+
+        cy.getByTestSubj('endpointPolicyForm-malware-blocklist-enableDisableSwitch').as(
```

Comment:
I would move `endpointPolicyForm-malware-blocklist-enableDisableSwitch` to a regular const, don't see any value in using cy vars here.